src/app/services/token.service.ts

```diff
@@ -0,0 +1,55 @@
+import { Injectable } from '@angular/core';
+
+const TOKEN_KEY = 'AuthToken';
+const USERNAME_KEY = 'AuthUserName';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
const USERNAME_KEY = "AuthUserName";
```